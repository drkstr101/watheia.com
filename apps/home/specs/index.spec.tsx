import React from 'react';
import { render } from '@testing-library/react';

import Index from '../pages/index';

describe('Index', () => {
  it('SHOULD render successfully', () => {
    const { container } = render(<Index />);
    expect(container).toBeInstanceOf(HTMLElement);
    expect(container).toMatchSnapshot();
  });

  // it('SHOULD auto-focus name input field', () => {
  //   const { getByTestId } = render(<Index />);
  //   const input = getByTestId('wa-hero-name-input');

  //   expect(input).toHaveFocus();
  // });
});
