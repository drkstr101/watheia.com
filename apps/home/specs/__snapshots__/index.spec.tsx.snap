// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index SHOULD render successfully 1`] = `<div />`;
